.prettierrc.json

```diff
@@ -0,0 +1,3 @@
+{
+  "singleQuote": true
```

Comment:
My understanding was that Prettier is opinionated and doesn't really encourage much user configuration. I ask the following question from a place of ignorance and because I couldn't easily figure out from the docs & Googling:
What does this do and why does it need to be here?

Reply:
This line certainly doesn't need to be here and reflects a personal preference. Its effect is to format JavaScript string literals using single quotes, with double quotes being the alternative.

Reply:
makes sense, I'm fine with that :thumbsup: